config.js

```diff
@@ -1,5 +1,10 @@
+const {
+  REQUEST_THROTTLE_MS,
+  REQUEST_TIMEOUT_MS,
+} = process.env;
+
 export default {
-  REQUEST_THROTTLE_MS: 5000,
-  REQUEST_TIMEOUT_MS: 5000,
+  REQUEST_THROTTLE_MS: Number(REQUEST_THROTTLE_MS) || 5000,
+  REQUEST_TIMEOUT_MS: Number(REQUEST_TIMEOUT_MS) || 5000,
```

Comment:
Yeah we should expect increased memory overhead, but I'm not sure how bad. Two places I can see memory being held with longer throttles/timeout would be in the [throttled requests we're keeping track of](https://github.com/artsy/metaphysics/blob/master/lib/throttle.js#L3) and any number of things certain closures might be holding on to during the lifecycle of [an outgoing request](https://github.com/artsy/metaphysics/blob/622bf2ac077a10cd5992c54f920fe790c72beb83/lib/loaders/http.js).

The former is just a bunch of booleans so I don't imagine that'll be much of a problem. The latter is hard to tell until we configure and measure it's impact. With increased `REQUEST_TIMEOUT_MS` I'd expect more memory usage as libraries/code tries to hold the state of a request longer. With increased `REQUEST_THROTTLE_MS` I'd actually expect less memory overhead and we more frequently resolve the request from cache and skip the background refreshing more often.